# configuration/exceptions.py

class DatabaseCredentialsError(Exception):
    pass
